### ISSUE TITLE
Multicolor for texts

### ISSUE THREAD

**Issue by 0PandaDEV** [holxsam/tldraw-in-obsidian#39]
It would be useful to have a popup every time you select a piece of text where you can then choose different formatting options, not only color changes. But I mainly need the different colors for the same text function.

**Comment by 0PandaDEV** [comment]
Yes something like that but instead of changingg it for the entire text filed it only changes for the specific selected part inside the text field

**Comment by jon-dez** [comment]
In the short term, I think implementing the color tool will be easier. The reason for this is because it doesn't seem that the base Tldraw text shapes allow for multiple styles inside their text components, or at least there seems to be no easy way. Due to these limitations, implementing custom components seems to be the only viable path since the entire visual part of a component can be controlled. The tricky part would probably be ensuring that the custom components can render in another Tldraw instance (e.g. https://tldraw.com) when loading the imported file so that it resembles how it looks like in Obsidian. This tricky part that I mentioned of course is not a required feature, but I think it would be neat to have.

**Comment by 0PandaDEV** [comment]
I already made a second issue on the repo of tldraw they said it has to do with rich text format so it's an implementation on their side.
